src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -96,6 +96,9 @@
          "mode" (type 's')
             One of auto, none, check. Defaults to auto, which follows configured
             policy (available in AutomaticUpdatePolicy property).
+         "redirect-output" (type 'b')
```

Comment:
This is a bit of a bikeshed...I'm OK keeping this as is but WDYT about calling this e.g. `output-to-self`?  `redirect-output` is more ambiguous.

Reply:
Done in fixup! :arrow_down: